apps/cde-ui/src/app/pages/create-visualization-page/create-visualization-page.component.ts

```diff
@@ -188,16 +170,25 @@ export class CreateVisualizationPageComponent {
     return this.customColorMapLoadError ? this.formatErrorMessage(this.customColorMapLoadError) : '';
   }
 
+  /** Error instructions for node upload error */
+  get nodesErrorInstructions(): string {
```

Comment:
Major duplication between the two getters. Refactor to a private helper function:
```ts
private getErrorActionMessage(error: ExtendedFileLoadError | undefined, fileDescription: string): string {
  switch (error?.type) {
    case undefined:
      return '';
    case 'type-error':
      return `Please upload a ${fileDescription} table CSV file.`;
    default:
      return 'Please upload a CSV file with the required columns.';
}

get nodesErrorActionMessage(): string {
  return this.getErrorActionMessage(this.nodesLoadError, 'cell type table')
}
```